src/makemsix/main.cpp

```diff
@@ -71,6 +77,12 @@ struct State
         return true;
     }
 
+    bool SkipAll()
```

Comment:
>SkipAll [](start = 9, length = 7)

This should be name more accordingly to applicability (aka SkipAllApplicability/SkipApplicability). SkipAll sounds, to me, that is a union of all the Skip* methods this struct has, which isn't true